LEGO1/omni/include/mxtimer.h

```diff
@@ -20,10 +20,18 @@ class MxTimer : public MxCore {
 			return g_lastTimeTimerStarted;
 		}
 		else {
-			return g_lastTimeCalculated - this->m_startTime;
+			return GetTimeElapsed();
 		}
 	}
 
+	MxLong GetStartTime() { return m_startTime; }
```

Comment:
These all appear to be unused? If yes, please remove

Reply:
Oh, my bad - these were left over from an earlier version where I had overlooked `GetTime()`.